stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -107,8 +110,8 @@ public class MessageComposerController(
      */
     public val channelState: Flow<ChannelState> = chatClient.watchChannelAsState(
         cid = channelId,
-        messageLimit = DefaultMessageLimit,
-        coroutineScope = scope
+        messageLimit = if (messageId != null) 0 else DefaultMessageLimit,
```

Comment:
Good, so in case no message is focused, then composer and message list will have the very same request which will be executed only once?

Reply:
Correct